flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -294,6 +294,13 @@
 				"they doesn't need to be downloaded every time for each application. An example could be " +
 				"hdfs://$namenode_address/path/of/flink/lib");
 
+	public static final ConfigOption<List<String>> YARN_ACCESS =
+		key("yarn.access.hadoopFileSystems")
```

Comment:
I believe it would be better to use `yarn.security.kerberos.additionalFileSystems`, because we have already a config key prefixed with `yarn.security.kerberos.` in the `yarn.` keys.

Reply:
thanks for your suggestion，I will change it.

> I believe it would be good to look into covering this change in our Kerberos end to end tests, to guarantee this is properly working.
> 
> You should be able to extend the end to end test defined in `test_yarn_job_kerberos_docker.sh` for this.
> You will probably need to set up a secondary namenode configuration in flink-end-to-end-tests/test-scripts/docker-hadoop-secure-cluster/config, create additional keys in `bootstrap.sh` and launch a second nameNode there.

okay,I will follow your advice.

> I believe it would be good to look into covering this change in our Kerberos end to end tests, to guarantee this is properly working.
> 
> You should be able to extend the end to end test defined in `test_yarn_job_kerberos_docker.sh` for this.
> You will probably need to set up a secondary namenode configuration in flink-end-to-end-tests/test-scripts/docker-hadoop-secure-cluster/config, create additional keys in `bootstrap.sh` and launch a second nameNode there.

okay,I will follow your advice. Thanks!